config.js

```diff
@@ -534,128 +534,128 @@ exports.updateFromArgv = function() {
     }
 
     if (argv.boardsize) {
-        for (let boardsize of argv.boardsize.split(',')) {
-            if (boardsize == "all") {
+        for (let e of argv.boardsize.split(',')) {
+            if (e === "all") {
                 exports.allow_all_sizes = true;
-            } else if (boardsize == "custom") {
+            } else if (e === "custom") {
                 exports.allow_custom_sizes = true;
-                for (let boardsizewidth of argv.boardsizewidth.split(',')) {
-                    exports.allowed_custom_boardsizewidth[boardsizewidth] = true;
+                for (let e of argv.boardsizewidth.split(',')) {
+                    exports.allowed_custom_boardsizewidth[e] = true;
                 }
-                for (let boardsizeheight of argv.boardsizeheight.split(',')) {
-                    exports.allowed_custom_boardsizeheight[boardsizeheight] = true;
+                for (let e of argv.boardsizeheight.split(',')) {
+                    exports.allowed_custom_boardsizeheight[e] = true;
                 }
             } else {
-                exports.allowed_sizes[boardsize] = true;
+                exports.allowed_sizes[e] = true;
             }
         }
     }
 
     if (argv.boardsizeranked) {
-        for (let boardsizeranked of argv.boardsizeranked.split(',')) {
-            if (boardsizeranked == "all") {
+        for (let e of argv.boardsizeranked.split(',')) {
```

Comment:
oh
why was the original one better ?

i thought it was faster and clearer to write `e` everytime instead of copypasting 3 different values for general/ranked/unranked

Reply:
You could also go with `size` or `boardsize` to make it clearer without having to rename each time you copy paste. Another logical thing to do if you are copy-pasting something 3 times, is to make it a function.

That said, if this is the result of a copy paste it's probably easier to stick with `e`, however you should not go out of your way to deliberately rename proper named variables to `e`.

It's all the balance of code hygiene versus getting things done quickly. You can be a little lazy and use a variable named `e` here and there, but once you have `e1`, `e2`, `i`, `j` and `p`, you probably wend way ahead of yourself and should go back to name them properly. Just like you cannot spend 15 hours a day washing yourself, you should probably also not go a month without a shower :-)

Reply:
ok lol
this is clear with the shower example :rofl:

Reply:
aahh gotta rebase all this lol now